projects/storefrontlib/src/lib/checkout/checkout.module.ts

```diff
@@ -10,12 +10,14 @@ import { metaReducers } from './store/reducers/index';
 import { services } from './facade/index';
 
 import { MultiStepCheckoutModule } from './components/multi-step-checkout/multi-step-checkout.module';
+import { CartModule } from '@spartacus/core';
```

Comment:
Why do we depend on CartModule in the checkout module? we should do it from Cart(Component)Module, but not from checkout module, unless there's a good reason.